contracts/Strategy.sol

```diff
@@ -288,11 +188,9 @@ contract Strategy is BaseStrategy {
                 amountToFree = deposited;
             }
             if (deposited > 0) {
-                //Claim rewards then withdraw
-                if(!emergencyExit)
-                    ISynthetixRewards(staker).getReward();
-                if(amountToFree > 0)
-                    ISynthetixRewards(staker).withdraw(amountToFree);
```

Comment:
Why remove emergencyexit conditional execution? we may need to call emergencywithdraw if there are issues with rewards not being given,unless this isnt a issue with the way ice has setup their farming contract

Reply:
withdrawal takes care of the reward, it's not like master chef.